src/utils.cpp

```diff
@@ -438,3 +439,33 @@ std::vector<std::string> Utils::Tokenize(const std::string &str_to_tokenize, con
 
 	return tokens;
 }
+
+
+std::string Utils::ReplacePlaceholders(const std::string& text_template, std::vector<char> types, std::vector<std::string> values) {
+	std::string str = text_template;
```

Comment:
Thanks! I’ll fix this in less than an hour. Looks like I’m increasing the variable by 2 (skipping %%) when I should be increasing in by 1 (skipping %).